### ISSUE TITLE
World Models TensorFlow implementation to reproduce experiments

### ISSUE THREAD

**Issue by hardmaru** [worldmodels/worldmodels.github.io#9]
I wrote a short blog post with step-by-step instructions for reproducing the experiments in the world models paper. http://blog.otoro.net/2018/06/09/world-models-experiments/

**Comment by kessler-frost** [comment]
It is written that it will take 6-8 hours for the whole process of executing gpu_bash.bash but it's been 12 hours and the training of van_train.py model hasn't been finished yet even on a 4 P100 GPU system with 220 GB of RAM. I guess you might need to modify some parameters, maybe epoch size or learning rate in the van_train.py file, in order to make it converge early. Great post though!
